packages/amplify-category-api/src/force-updates/api-resource-paths.ts

```diff
@@ -0,0 +1,49 @@
+import { pathManager, stateManager } from '@aws-amplify/amplify-cli-core';
```

Comment:
This file is just the existing functions from `auth-notifications.ts` which pertained to getting the api resource path.